gtk4/src/auto/tree_model.rs

```diff
@@ -56,7 +53,7 @@ pub trait TreeModelExt: 'static {
     fn get_string_from_iter(&self, iter: &TreeIter) -> Option<glib::GString>;
 
     #[doc(alias = "gtk_tree_model_get_value")]
-    fn get_value(&self, iter: &TreeIter, column: i32) -> glib::Value;
+    fn get(&self, iter: &TreeIter, column: i32) -> glib::Value;
```

Comment:
Why is this not nullable? If you pass in a wrong column you surely get back `NULL` or something? :)

Reply:
Seems like the C function doesn't return a `gboolean` to tell whether the value was actually set to something or not. It probably needs at least a check whether `column < get_n_column()`

Reply:
Does it also need the `GValue` to be initialized with the correct type before passing a mutable reference to the C function?

Reply:
From the use cases I found in the code, it only gets passed a `G_VALUE_INIT`

Reply:
Added an assertion for `column < get_n_column()`